README.md

```diff
@@ -546,6 +546,8 @@ console.log(storeInstance.selectedTodo.title)
 // prints "Get coffee"
 ```
 
+For a custom-
+
```

Comment:
this probably isn't supposed to be here :)

Reply:
sorry this PR should have been closed...

(it's related to #558 - others in the issue didn't like the idea of exporting CoreType)

though I'd really appreciate if someone would implement custom-primitive support...